From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Johnson <johnson.peter@gmail.com>
Date: Sat, 10 Jun 2023 14:41:39 -0700
Subject: [PATCH 2/9] Remove redundant move

---
 src/google/protobuf/extension_set.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/google/protobuf/extension_set.h b/src/google/protobuf/extension_set.h
index 5cfd5727b19e89dc1511c3891ea1145e79ede3a9..cf9383a78fb8c322a84a927d6c3c968d6d00cf3d 100644
--- a/src/google/protobuf/extension_set.h
+++ b/src/google/protobuf/extension_set.h
@@ -726,7 +726,7 @@ class PROTOBUF_EXPORT ExtensionSet {
   static KeyValueFunctor ForEach(Iterator begin, Iterator end,
                                  KeyValueFunctor func) {
     for (Iterator it = begin; it != end; ++it) func(it->first, it->second);
-    return std::move(func);
+    return func;
   }
 
   // Applies a functor to the <int, Extension&> pairs in sorted order.
