From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Johnson <johnson.peter@gmail.com>
Date: Sat, 10 Jun 2023 14:41:39 -0700
Subject: [PATCH 02/10] Remove redundant move

---
 src/google/protobuf/extension_set.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/google/protobuf/extension_set.h b/src/google/protobuf/extension_set.h
index e85370a412ccb9557986525516cc1acfae2452ad..583a735aac277d488ce77e7ba01aa9802b891f3f 100644
--- a/src/google/protobuf/extension_set.h
+++ b/src/google/protobuf/extension_set.h
@@ -752,7 +752,7 @@ class PROTOBUF_EXPORT ExtensionSet {
   static KeyValueFunctor ForEach(Iterator begin, Iterator end,
                                  KeyValueFunctor func) {
     for (Iterator it = begin; it != end; ++it) func(it->first, it->second);
-    return std::move(func);
+    return func;
   }
 
   // Applies a functor to the <int, Extension&> pairs in sorted order.
